hw20/group_vars/centos/centos.yml

```diff
@@ -0,0 +1,5 @@
+---
+#Centos vars
+ansible_user: vagrant
```

Comment:
`ansible_user` and `ansible_ssh_private_key_file` should be in general var group as you have same values.

Reply:
You can leave it only in `linux` or create `all`